nginx.conf

```diff
@@ -20,13 +24,13 @@ http {
 
     server {
         listen ${{PORT}};
-        listen ${{SSL_PORT}} ssl;
-#        ssl_certificate     www.example.com.crt;
-#        ssl_certificate_key www.example.com.key;
-        ssl_certificate     host.cert;
-        ssl_certificate_key host.key;
+        # listen ${{SSL_PORT}} ssl;
+        server_name ${{HOSTNAME}};
+
+        # ssl_certificate     ${{SSL_CERT_NAME}}.cert;
+        # ssl_certificate_key ${{SSL_CERT_NAME}}.key;
```

Comment:
These are totally necessary as fallback certificates, as advised by the author of lua-resty-auto-ssl.